src/client/package.json

```diff
@@ -3,13 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
     "@testing-library/jest-dom": "^5.16.5",
```

Comment:
You can use the imports from fontawesome instead of adding svg files.
These lines would go at the top of the jsx:
```
import { faMagnifyingGlass} from '@fortawesome/free-solid-svg-icons'
import { faCaretRight} from '@fortawesome/free-solid-svg-icons'
import { faCaretLeft} from '@fortawesome/free-solid-svg-icons'
```
and then you can use them as:
`<FontAwesomeIcon icon={faCaretLeft} />`
directly in your code. You can style them as well with color, size, position.